.connector-store/meta.json

```diff
@@ -18,6 +23,203 @@
+                    "isHidden":false
+                },
+                {
+                    "name":"getJobInfo",
+                    "description":"An operation to get job info in Salesforce using bulk api 2.0",
+                    "isHidden":false
+                },
+                {
+                    "name":"deleteJob",
+                    "description":"An operation to delete a job in Salesforce using bulk api 2.0",
+                    "isHidden":false
+                },
+                {
+                    "name":"getSuccessfulResults",
+                    "description":"An operation to get successful results of a job in Salesforce using bulk api 2.0",
+                    "isHidden":false
+                },
+                {
+                    "name":"getUnprocessedResults",
+                    "description":"An operation to get unprocessed results of a job in Salesforce using bulk api 2.0",
+                    "isHidden":false
+                },
+                {
+                    "name":"getFailedResults",
+                    "description":"An operation to get failed results of a job in Salesforce using bulk api 2.0",
+                    "isHidden":false
+                },
+                {
+                    "name":"abortQueryJob",
+                    "description":"An operation to abort a running query job in Salesforce using bulk api 2.0",
+                    "isHidden":false
+                },
+                {
+                    "name":"createQueryJob",
+                    "description":"An operation to create a new query job in Salesforce using bulk api 2.0",
+                    "isHidden":false
+                },
+                {
+                    "name":"deleteQueryJob",
+                    "description":"An operation to delete a query job in Salesforce using bulk api 2.0",
+                    "isHidden":false
+                },
+                {
+                    "name":"getQueryJobInfo",
+                    "description":"An operation to get information about a query job in Salesforce using bulk api 2.0",
+                    "isHidden":false
+                },
+                {
+                    "name":"getQueryJobResults",
+                    "description":"An operation to get the results of a query job in Salesforce using bulk api 2.0",
+                    "isHidden":false
+                },
+                {
+                    "name":"getAllQueryJobInfo",
+                    "description":"An operation to get the information of all the query jobs in Salesforce using bulk api 2.0",
+                    "isHidden":false
+                }
+            ],
+            "connections": [
+                {
+                    "name": "salesforcebulk",
+                    "description": "Connection for bulk data operations in Salesforce."
+                }
+            ],
+            "isHidden": false
+        },
```

Comment:
There are quite a few releases listed here: https://github.com/wso2-extensions/esb-connector-salesforcebulk/tags. is there any particular reason only 1.0.7 was added?

Reply:
We use the latest two releases for the two salesforce versions. We can stick with that.